src/api/json/catalog.json

```diff
@@ -1634,6 +1634,12 @@
       "fileMatch": ["coffeelint.json"],
       "url": "https://json.schemastore.org/coffeelint.json"
     },
+    {
+      "name": "committed.toml",
+      "description": "committed configuration file",
```

Comment:
Spelled this intentionally in lowercase, that's how upstream refers to themselves quite consistently, too. Can obviously change to titlecase in case it's preferred here regardless.

Reply:
Sounds good